db-models/listings.sql

```diff
@@ -0,0 +1,14 @@
+-- db-models/listings.sql
+CREATE TABLE listings (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,       -- Auto-incrementing ID
+    author TEXT NOT NULL,                       
+    title TEXT NOT NULL,                     
+    price  NOT NULL,                            -- Floating-point number
```

Comment:
`price` has no data type currently, I think in person you showed it to be `REAL?`